core/domain/opportunity_services.py

```diff
@@ -265,16 +287,23 @@ def _create_exploration_opportunities(
             create_exp_opportunity_summary(
                 topic, story, exploration))
     _save_multi_exploration_opportunity_summary(
-        exploration_opportunity_summary_list)
+        exploration_opportunity_summary_list
+    )
 
 
-def update_opportunity_with_updated_exploration(exp_id: str) -> None:
+def update_opportunity_models_with_updated_exploration(
```

Comment:
Same thing, here and in the docstring: "update models" means update the datastore. Make sure the names and docstrings are not misleading.